### ISSUE TITLE
Remove "Hide Responsive Voices" by default and only show when "Show all voices" are activated

### ISSUE THREAD

**Issue by arasaac-dga** [asterics/AsTeRICS-Grid#181]
We have several issues with users that consider that responsive voices are native or are offered with Asterics Grid and that, after select some of them, generate problems because they need internet (an many of them work without Internet to avoid children navigate to some pages) or the speak happen with delay some seconds after select a cell. 

So we propose to improve usability and avoid confusions:

1) "Hide Responsive Voices" by default and only show when "Show all voices" are activated
2) Add after the responsive voice name a "(need Internet)" or something similar
3) (optional) Perhaps if there is no native voice in the device show a message that advice of the need of set a TTS Engine in the device configuration or buy some voice (without name any commercial). 

The main confusion is, in our experience, with tablets (Huawei, Lenovo, ....) that not offer TTS Voices for free as make Samsung ones (for some languages).

**Comment by ChrisVeigl** [comment]
It's a good idea to make the options for local (native) and online voices better understandable!
It's an important and difficult topic - considering the different situations on various operating systems ... 
see also initial comment here: https://github.com/asterics/AsTeRICS-Grid/issues/158
I think we need a seperate video tutorial for this and/or instructions how to install native voices on different OSes. Maybe also tips for affordable commercial voices / kid's voices etc.
Could we link to such tutorials directly from the AsTeRICS grid app (?-button next to critical setttings) ?

**Comment by klues** [comment]
Yes, I also think that it should be clearly distinguishable which voices are working online and which offline. I've just found out that there is a property in the API that can be used for that, see [MDN documentation for localService](https://developer.mozilla.org/en-US/docs/Web/API/SpeechSynthesisVoice/localService).

I think local voices should be listed first and if the user selects an online voice maybe there should be some warning that it's only working online with a link to further documentation where it's described how to install other voices. I would keep it as it is to filter voices by current content language and not hiding online voices by default, since it would also hide all Google voices in Chrome and Microsoft voices in Edge, which are normally working very well.

> Could we link to such tutorials directly from the AsTeRICS grid app (?-button next to critical setttings) ?

I think I would link to a documentation page (either our manual or the ARASAAC tutorial), which then can link to various other ressources or videos. But yes, of course it's possible to add a ?-button, or a link in a text.

**Comment by arasaac-dga** [comment]
Ok. We consider that the Automatic option must be selected always by default and use, by default, the first local/native voice that is available. Local voices must be listed first as you comment and, in case of selecting an online voice, (specially in Android), if it's possible, advice of the possible problems (delays), the need of internet connection and the recommendation of use local/native voices. We can make a tutorial specially focused on Android (where there are most of the problems) explaining how to set a TTS Engine. Only we don't use to recommend, in public, commercial voices. Only when somebody asks us or in courses we recommend some commercial voice that we use and have experience with.
